thumbprint/src/main/java/com/thumbtack/thumbprint/views/edittext/ThumbprintEditTextBase.kt

```diff
@@ -65,7 +67,9 @@ abstract class ThumbprintEditTextBase(
                 ) {
                 }
 
-                override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {}
+                override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
+                    hasError = errorValidator?.invoke(s) == true
```

Comment:
Actually, any thought compared to this logic?
```
hasError = if (errorValidator != null) {
    errorValidator?.invoke(s) == true
} else {
    hasError
}
```
I'm thinking maybe we should deprecate hasError and rely on hasErrorIf